src/nft/components/collection/CollectionStats.tsx

```diff
@@ -338,18 +339,28 @@ const statsLoadingSkeleton = (isMobile: boolean) =>
   ))
 
 const StatsRow = ({ stats, isMobile, ...props }: { stats: GenieCollection; isMobile?: boolean } & BoxProps) => {
+  const { formatNumberOrString } = useFormatter()
+
   const uniqueOwnersPercentage = stats?.stats?.total_supply
     ? roundWholePercentage(((stats.stats.num_owners ?? 0) / stats.stats.total_supply) * 100)
     : 0
-  const totalSupplyStr = stats.stats ? quantityFormatter(stats.stats.total_supply ?? 0) : 0
+  const totalSupplyStr = stats.stats
+    ? formatNumberOrString({ input: stats.stats.total_supply ?? 0, type: NumberType.NFTCollectionStats })
+    : 0
   const listedPercentageStr = stats?.stats?.total_supply
     ? roundWholePercentage(((stats.stats.total_listings ?? 0) / stats.stats.total_supply) * 100)
     : 0
   const isCollectionStatsLoading = useIsCollectionLoading((state) => state.isCollectionStatsLoading)
 
   // round daily volume & floorPrice to 3 decimals or less
-  const totalVolumeStr = volumeFormatter(Number(stats.stats?.total_volume) ?? 0)
-  const floorPriceStr = floorFormatter(stats.stats?.floor_price ?? 0)
+  const totalVolumeStr = formatNumberOrString({
+    input: Number(stats.stats?.total_volume) ?? 0,
+    type: NumberType.NFTCollectionStats,
+  })
+  const floorPriceStr = formatNumberOrString({
+    input: stats.stats?.floor_price ?? 0,
+    type: NumberType.NFTTokenFloorPrice,
+  })
   // graphQL formatted %age values out of 100, whereas v3 endpoint did a decimal between 0 & 1
   const floorChangeStr = Math.round(Math.abs(stats?.stats?.one_day_floor_change ?? 0))
```

Comment:
`uniqueOwnersPercentage` and `listedPercentageStr` probably do not need since there values will always be between 0-100, but `floorChangeStr` could potentially be much higher and should take into account locale formatting?